### PR TITLE
[TASK] add link type as title in tables

### PR DESCRIPTION
**Pull request by genofire**

### CODE REVIEW

File: lib/infobox/node.js [hunk 1]
```diff
@@ -61,9 +61,9 @@ define(['sorttable', 'snabbdom', 'd3-interpolate', 'moment', 'helper', 'utils/no
       }
 
       function renderNeighbourRow(n) {
-        var icons = '';
+        var icons = [V.h('span', { props: { className: 'icon ion-' + (n.link.type === 'wifi' ? 'wifi' : 'share-alt'), title: _.t(n.link.type) } })];
```
Comment (genofire) [inline lib/infobox/node.js]: Maybe usw `link.indexOf('wifi') === 0` for later implementation of different wifi types like `wifi24` and `wifi5`.

File: lib/infobox/node.js [hunk 2]
```diff
@@ -61,9 +61,9 @@ define(['sorttable', 'snabbdom', 'd3-interpolate', 'moment', 'helper', 'utils/no
       }
 
       function renderNeighbourRow(n) {
-        var icons = '';
+        var icons = [V.h('span', { props: { className: 'icon ion-' + (n.link.type.indexOf('wifi') ? 'wifi' : 'share-alt'), title: _.t(n.link.type) } })];
```
Comment (genofire) [inline lib/infobox/node.js:64]: Da fehlt das `=== 0`

Comment (xf-) [reply lib/infobox/node.js:64]: das is drin ;)

Comment (genofire) [reply lib/infobox/node.js:64]: My fault

**Pull request merged**